src/components/SiteSearch.vue

```diff
@@ -103,13 +103,11 @@ export default class SiteSearch extends Vue {
   private resultLimit: Number = 7;
   private fuse: any = null;
   private value: String = "";
-  private quickLinkLoc: any = [];
   private keyEvents: any = [];
   private currentResult: number = 0;
 
   @Prop()
   jsonSearch!: any;
-  searchData = this.jsonSearch;
```

Comment:
Basically...let's copy jsonSearch. Do nothing to mutate the data. And then watch the copied data that never changes for changes. Why not just watch jsonSearch instead?

Action: replace searchData with jsonSearch everywhere

Reply:
I was working on migrating beaker to Vue 2.7 and as I went through this file, this was my concern as well.